next-packages/serialize/src/index.js

```diff
@@ -173,15 +173,15 @@ function createStringFromObject(
       string += handleInterpolation(mergedProps, registered, obj[i], false)
     }
   } else {
-    for (let key in obj) {
-      if (typeof obj[key] !== 'object') {
-        if (registered[obj[key]] !== undefined) {
-          string += `${key}{${registered[obj[key]]}}`
+    // using var instead of let here because using let here makes babel use a closure
```

Comment:
not entirely sure, but seems like a deopt in babel that could be fixed, gonna try to debug this later, this causes it:
```js
          value.forEach(value => {
            string += `${processStyleName(key)}:${processStyleValue(
              key,
              value
            )};`
          })
```
but original `value` is not referenced inside the callback so I don't see why would it deopt because of this